dsgrid/dataset/dataset_schema_handler_standard.py

```diff
@@ -60,6 +60,7 @@ def get_unique_dimension_rows(self):
     def make_project_dataframe(self):
         # TODO: Can we remove NULLs at registration time?
         lk_df = self._load_data_lookup.filter("id is not NULL")
+        lk_df = self._add_time_zone(lk_df)
```

Comment:
Per our informal discussion today, we need something defined at this level that tells us whether we are delivering time as experienced by people in specific time zones or as absolute time. It will change how we perform aggregations.